packages/react/package.json

```diff
@@ -30,6 +30,7 @@
     "@cds/core": "./dist/core"
   },
   "devDependencies": {
+    "@cds/core": "./dist/core",
```

Comment:
The core package was missing in the `node_modules` cause the package is set a a `peerDep` and to run the dev app you have to install it manually. 

Added it as a devDep to enable running the dev app